### PR TITLE
Added Info about the difference between workflowId & triggerId [DVRL-25]

### PR DESCRIPTION
**Pull request by iampearceman**
Screenshot 2023-11-27 at 1 55 51

### CODE REVIEW

File: workflows/notification-workflows.mdx [hunk 1]
```diff
@@ -92,4 +92,20 @@ When all channels are turned off for that subscriber then subscriber will not re
...
 </Accordion>
 
+<Accordion title="What is the difference between workflowId and trigger identifier?">
+`workflowId` uniquely identifies each workflow in the Novu ecosystem. When you create a workflow, it encapsulates the entire flow and rules of all messages that are being sent. It includes various components such as trigger, channels, etc.
+<img
+  style={{ borderRadius: '0.5rem' }}
+  src="/images/workflows/workflow-id.png"
+/>
+
+`trigger identifier`, on the other hand, refers to an event or action that initiates the workflow. It serves as the start point which sets the entire workflow into motion. Whenever an event takes place in your system that matches a trigger identifier, Novu gets the signal to start the specified workflows.
```
Comment (jainpawan21) [inline workflows/notification-workflows.mdx:102]: I am not sure this information is correct

Comment (jainpawan21) [reply workflows/notification-workflows.mdx:102]: @unicodeveloper check above comment

Comment (unicodeveloper) [reply workflows/notification-workflows.mdx:102]: That's very interesting. Thanks for the heads up @jainpawan21.

**Comment by unicodeveloper** [comment]
@iampearceman Why is there still a delay here?

File: workflows/notification-workflows.mdx (hunk 1 above)
Comment (iampearceman) [reply workflows/notification-workflows.mdx]: Done

**Review by unicodeveloper — APPROVED** [review]

**Pull request merged**